cardano/src/coin.rs

```diff
@@ -100,6 +100,10 @@ impl Coin {
             Ordering::Less    => CoinDiff::Negative(Coin(against.0 - self.0)),
         }
     }
+
+    pub fn to_integral(self) -> u64 {
+        return self.0
+    }
```

Comment:
This function is not needed because of the presence of `Deref` with the `Target = u64`.